### PR TITLE
Homepage not linked from logo for screen readers

### PR DESCRIPTION
**Pull request by LouiaR**
Type: patch

Issue  is here: https://github.com/comicrelief/comicrelief-contentful/issues/485

### CODE REVIEW

File: src/components/Atoms/Logo/Logo.test.js [hunk 1]
```diff
@@ -32,12 +32,11 @@ it('renders correctly', () => {
     }
 
     <div
-      aria-label="logo"
       className="c0"
       rotate={1}
     >
       <img
-        alt="Comic Relief logo"
+        alt="Comic Relief homepage"
```
Comment (gusliedke) [inline src/components/Atoms/Logo/Logo.test.js]: Please change to `Comic Relief logo – home page`

File: src/components/Atoms/Logo/Logo.js [hunk 2]
```diff
@@ -37,7 +35,9 @@ const Logo = ({ rotate, sizeSm, sizeMd, campaign }) => {
       <Image
         src={campaign === 'comicrelief' ? crLogo : srLogo}
         alt={
-          campaign === 'comicrelief' ? 'Comic Relief logo' : 'Sport Relief logo'
+          campaign === 'comicrelief'
+            ? 'Comic Relief logo - home page'
```
Comment (gusliedke) [inline src/components/Atoms/Logo/Logo.js]: @LouiaR sorry, this was a recommendation from DAC report. But thinking better I don't think it make sense to use home page here.. this logo is also used in the footer. Let's just use `Comic Relief logo` and on CR we add `title="homepage"` to the link and a visually hidden span.

Comment (LouiaR) [reply src/components/Atoms/Logo/Logo.js]: @gusliedke what about add `alt='comicrelief.com'`

File: src/components/Molecules/Footer/Footer.js [hunk 3]
```diff
@@ -10,19 +10,27 @@ import {
...
 } from './Footer.style';
 
 const Footer = ({ navItems, footerCopy, campaign, ...rest }) => {
   return (
     <FooterWrapper navItems {...rest}>
       <InnerWrapper>
         <SocialIconWrapper>
-          <SocialIcons campaign={campaign} />
+          <SocialIcons
+            campaign={campaign
```
Comment (gusliedke) [inline src/components/Molecules/Footer/Footer.js]: Can you create a new const `campaignName` and use `.replace(/ +/g, "").join('').toLowerCase()` instead? So we can read what's doing.

File: src/components/Molecules/Footer/Footer.js [hunk 4]
```diff
@@ -10,19 +10,27 @@ import {
...
   FooterCopyright,
-  SocialIconWrapper
+  SocialIconWrapper,
+  Brand
 } from './Footer.style';
 
 const Footer = ({ navItems, footerCopy, campaign, ...rest }) => {
+  // Concatenate campaign in one string
+  const campaignName = campaign
+    .split(' ')
```
Comment (gusliedke) [inline src/components/Molecules/Footer/Footer.js]: Can you create a new const campaignName and use .replace(/ +/g, "").join('').toLowerCase() instead? So we can read what's doing.

File: src/components/Molecules/Footer/Footer.js [hunk 5]
```diff
@@ -10,19 +10,24 @@ import {
   InnerWrapper,
   FooterBranding,
   FooterCopyright,
-  SocialIconWrapper
+  SocialIconWrapper,
+  Brand
 } from './Footer.style';
 
 const Footer = ({ navItems, footerCopy, campaign, ...rest }) => {
+  // Concatenate campaign in one string
```
Comment (gusliedke) [inline src/components/Molecules/Footer/Footer.js]: Can u just replace to `Remove white scape between words`?

**Pull request merged**